runhouse/resources/hardware/cluster_factory.py

```diff
@@ -24,6 +28,15 @@ def cluster(
         host (str or List[str], optional): Hostname, IP address, or list of IP addresses for the BYO cluster.
         ssh_creds (dict, optional): Dictionary mapping SSH credentials.
             Example: ``ssh_creds={'ssh_user': '...', 'ssh_private_key':'<path_to_key>'}``
+        server_port (bool, optional): Port to use for the server. (Default: ``50052``).
```

Comment:
flagging both lines for a potential change given our conversation earlier today.